src/pydata_sphinx_theme/__init__.py

```diff
@@ -1084,6 +1085,43 @@ def setup_translators(app):
             app.set_translator(name, translator, override=True)
 
 
+# ------------------------------------------------------------------------------
+# customize method for logo management
+# ------------------------------------------------------------------------------
+
+
+def setup_logo_path(
+    app: Sphinx, pagename: str, templatename: str, context: dict, doctree: nodes.Node
+) -> None:
+    """Set up relative paths to logos independant from the version"""
+
+    # get informations from the context
+    pathto = context.get("pathto")
+    logo = context.get("logo")
+    image_light = context.get("theme_logo", {}).get("image_light")
+    image_dark = context.get("theme_logo", {}).get("image_dark")
+
+    # create theme_options["logo"] if not existing
+    if "theme_logo" not in context:
+        context["theme_logo"] = {}
```

Comment:
the comment doesn't quite match what the code is doing here.

Reply:
that's was coresponding to what I was doing before @choldgraf suggestions

Reply:
solved in https://github.com/pydata/pydata-sphinx-theme/pull/1132/commits/5d3fff74ec9232aa7451c30d5c73ed0f39c92000 ?